docs/src/components/Card/card.module.css

```
@@ -131,3 +131,35 @@
   max-height: 80px;
   justify-content: center;
 }
+
+.NewFeatureCardContent {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.NewFeatureCardHeading {
+  color: var(--ifm-color-primary-darkest);
+  font-size: 1.75rem;
```

could we reduce this to 1.5rem?
```suggestion
  font-size: 1.5rem;
```